introduction.tex

```diff
@@ -62,6 +62,9 @@ \subsection{Register Definition Format}
         \hline
         R/W1 & Read/Write. Only writing 1 has an effect.  \\
         \hline
+        R/W1C & Read/Write. Writing every bit high clears the field. Writing
```

Comment:
Hmm... good point. But other fields (eg. cmderr) contain a multi-bit number. If you clear them bit-by-bit there's a race. Eg. the register contains 2. Debugger writes 2 to clear it, but error 3 happens just before. Now the next time the register is read it contains 1.
Actually in the case of cmderr that can't happen since the errors are sticky.

I think it makes sense to define fields that are really bitfields themselves as individual fields. Then this description is correct.

Thoughts?